config/canonical.json

```diff
@@ -32616,8 +32616,10 @@
   "shop/travel_agency|Thomson": {
     "count": 113,
     "tags": {
-      "brand": "Thomson",
-      "name": "Thomson",
+      "brand": "The Thompson Travel Agency",
+      "brand:wikidata": "Q1170162",
+      "brand:wikipedia": "en:The Thompson Travel Agency",
+      "name": "The Thompson Travel Agency",
```

Comment:
Did you really read the [wikidata entry](https://www.wikidata.org/wiki/Q1170162)? This links to a book written by Jules Verne but not to a travel agancy called 'Thomson'... 😄
So please change back the `name` and the `brand` tag, and maybe change the `brand:wikipedia` tag to https://en.wikipedia.org/wiki/TUI_UK (if that is the correct entry...) and the `brand:wikidata` tag to [Q7795876](https://www.wikidata.org/wiki/Q7795876) (if that is the correct entry...)

Reply:
what exactly should I put into `brand:wikipedia`? Not the link right?

Reply:
i committed a change, but i'm not sure it's correct